site/scripts/initialise-shell-variables.sh

```diff
@@ -12,6 +12,7 @@ if [[ "true" == "${PROTECTED}" ]] ; then
   export SP_CERTIFICATE=`base64 .secrets/sp/sp.certificate`
   export SP_KEY=`base64 .secrets/sp/sp.key`
   export SP_CALLBACK_URL="http://${SP_DOMAIN_NAME}/sso/callback"
+  export UPLOADCARE_SECRET_KEY="${UPLOADCARE_SECRET_KEY}"
```

Comment:
Surprised this is the only variable added here? There were quite a few defined.

Reply:
updated